cmd/reviewdog/main.go

```diff
@@ -329,6 +351,45 @@ github-pr-check reporter as a fallback.
 			return err
 		}
 		ds = d
+	case "bitbucket-code-report":
+		build, client, ct, err := bitbucketBuildWithClient(ctx)
+		if err != nil {
+			return err
+		}
+		ctx = ct
+
+		// get list of runners, so we can create "green" for that runner
+		// if there are no issues from it
+		runnersList := strings.Split(opt.runners, ",")
+		if projectConf != nil {
+			for runner := range projectConf.Runner {
+				runnersList = append(runnersList, runner)
+			}
+		}
+
```

Comment:
runnersList will contains duplicated name with this code.
`-runners` is for running specific tools in project configuration, so appending all projectConf.Runner with opt.runners results in duplicates.

Can you create a util function in main.go or project/ package to get the list of tools reviewdog will run?